source/extensions/tracers/xray/tracer.h

```diff
@@ -115,6 +118,27 @@ class Span : public Tracing::Span, Logger::Loggable<Logger::Id::config> {
     aws_metadata_ = aws_metadata;
   }
 
+  /*
+   * Adds to the http request annotation field of the Span.
+   */
+  void addToHttpRequestAnnotations(absl::string_view key, ProtobufWkt::Value value) {
```

Comment:
Should `ProtobufWkt::Value` be a reference?

Reply:
done